extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/DefaultHttpCoordinator.cs

```diff
@@ -43,8 +44,7 @@ public async Task<FunctionContext> SetHttpContextAsync(string invocationId, Http
 
         public async Task<HttpContext> SetFunctionContextAsync(string invocationId, FunctionContext context)
         {
-            var contextRef = _contextReferenceList.GetOrAdd(invocationId, static id => new ContextReference(id));
-            contextRef.SetCancellationToken(context.CancellationToken);
+            var contextRef = _contextReferenceList.GetOrAdd(invocationId, id => new ContextReference(id));
```

Comment:
Why was the `static` modifier removed?

Reply:
Leftovers from a different change that I forgot to undo, thanks for flagging!